lib/dal/src/pkg/export.rs

```diff
@@ -1435,42 +1175,21 @@ impl PkgExporter {
                 pkg_spec_builder.schemas(schemas);
             }
             SiPkgKind::WorkspaceBackup => {
-                // let (mut head_funcs, funcs, schemas, components, edges) =
-                //     self.export_change_set(ctx, Some(ChangeSetId::NONE)).await?;
-
-                // head_funcs.extend_from_slice(&funcs);
-
-                // let mut head_builder = ChangeSetSpec::builder();
-                // head_builder
-                //     .name("head")
-                //     .schemas(schemas)
-                //     .components(components)
-                //     .edges(edges);
-
-                // pkg_spec_builder.default_change_set("head");
-
-                // for change_set in ChangeSet::list_open(ctx).await? {
-                //     let (funcs, referenced_head_funcs, schemas, components, edges) =
-                //         self.export_change_set(ctx, Some(change_set.pk)).await?;
-                //     head_funcs.extend_from_slice(&referenced_head_funcs);
-
-                //     pkg_spec_builder.change_set(
-                //         ChangeSetSpec::builder()
-                //             .name(&change_set.name)
-                //             .based_on_change_set("head")
-                //             .funcs(remove_duplicate_func_specs(&funcs))
-                //             .schemas(schemas)
-                //             .components(components)
-                //             .edges(edges)
-                //             .build()?,
-                //     );
-                // }
-
-                // pkg_spec_builder.change_set(
-                //     head_builder
-                //         .funcs(remove_duplicate_func_specs(&head_funcs))
-                //         .build()?,
-                // );
+                info!("Workspace exports not supported");
```

Comment:
We should probably just fail here instead of returning an OK. 
adding a new PkgError variant that returns what the info! above does will do the trick

Reply:
ok! Happy to do that :)